style.css

```diff
@@ -7,7 +7,7 @@
 }
 @font-face{
 	font-family:'Simvoni';
-	src:url(./fonts/Simvoni.ttf) format("truetype")
+	src:url(./fonts/Simvoni-Bold.ttf) format("truetype")
```

Comment:
Titles on iOS now look awful thanks to this, but there is no patch because iPads appear as Macs.